src/main/cliapp/src/containers/layout/SiteLayout.js

```diff
@@ -340,6 +340,14 @@ export const SiteLayout = (props) => {
 				}
 		];
 
+		const sortedMenu = [...menu];
+		sortedMenu[0].items.forEach((item) => {
+			if (item.label.match("Data Tables")) {
+				item.items = item.items.sort((a, b) => a.label.localeCompare(b.label));
+			}
+		});
```

Comment:
It would probally be better to just hardcode the order rather than using a sort method. This way if the lists get rearranged or changed in some way there is not unexpected behavior.